.github/workflows/publish-versioned.yml

```diff
@@ -0,0 +1,52 @@
+name: publish_versioned_docs
+
+on:
+  push:
+    branches:
+    - 'release-*'
+    tags:
+    - 'v[0-9]+.[0-9]+.[0-9]+'
+    paths:
+    - 'docs/user/**'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+      - name: Build Website
+        run: |
+          docker run -v ${PWD}/sources:/hugo/content/docs/agent --rm grafana/docs-base:latest /bin/bash -c 'make hugo'
+  sync:
+    runs-on: ubuntu-latest
+    needs: test
+    steps:
+    - name: Checkout Actions library
+      uses: actions/checkout@v2
```

Comment:
Is there a reason to use `actions/checkout@v2` and not `actions/checkout@v3` as is elsewhere?

Reply:
Oops, just missed that, thanks.